process/project-lifecycle-documents/gittuf_sandbox_stage.md

```diff
@@ -76,8 +76,8 @@ badge.
 
 ### IP policy and licensing due diligence
 
-License due diligence is applicable. An issue will be opened alongside this
-application to track that process.
+License due diligence is applicable. We have requested the IP policy and license
+review from the OpenSSF [here](https://github.com/ossf/tac/issues/199).
```

Comment:
I'm not a member of the org, but from an [a11y](https://www.a11yproject.com/) perspective, [`here` links are considered harmful](https://granicus.com/blog/why-click-here-links-are-bad/#:~:text=Click%20Here%20Links%20are%20Bad%20for%20Accessibility&text=If%20all%20of%20your%20links,to%20establish%20context%20for%20links.)

Reply:
Thanks, addressed! [45ded85](https://github.com/ossf/tac/pull/198/commits/45ded85f3c3b7784b9ad6882feef35fc5ad0fbd3)